config/locales/en.yml

```diff
@@ -11,6 +11,7 @@ en:
       edit_service_provider: Edit
       trigger_idp_refresh: Publish service providers
     required_field: Indicates a required field.
+    confirm: 'Are you sure?'
```

Comment:
maybe "Are you sure you want to delete this service provider? This cannot be undone."